### ISSUE TITLE
pitch_ Angle and roll_ Angle data is displayed as 0

### ISSUE THREAD

**Issue by NanYuan-C** [d3ngit/djihdfpv_mavlink_to_msp_V2#10]
I don't know if your test is correct, thank you.

**Comment by d3ngit** [comment]
Yes correct, they're not implemented. The data is not even pulled out of mavlink. I can add it in when I do the next update.

**Comment by MR444** [comment]
Hello @d3ngit, do you still work on that great project? I don't know why but it looks like there is still a gap for digital FPV systems supporting MavLink.
